x-pack/plugins/serverless_search/public/application/components/index_management/connector_empty_prompt.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiButtonEmpty, EuiPanel } from '@elastic/eui';
+
+import { BACK_LABEL } from '../../../../common/i18n_string';
+import { EmptyConnectorsPrompt } from '../connectors/empty_connectors_prompt';
+
+interface ConnectorIndexEmptyPromptProps {
+  indexName: string;
+  onBackClick?: () => void;
+}
+
+export const ConnectorIndexEmptyPrompt = ({ onBackClick }: ConnectorIndexEmptyPromptProps) => {
+  return (
+    <EuiPanel>
+      <EuiButtonEmpty
+        data-test-subj="serverlessSearchConnectorIndexEmptyPromptBackButton"
+        onClick={onBackClick}
+        iconSide="left"
+        iconType="arrowLeft"
+      >
+        {BACK_LABEL}
+      </EuiButtonEmpty>
+      <EmptyConnectorsPrompt />
```

Comment:
@sphilipse importing and rendering this component is what is breaking the lazy loading, if I remove this and run metrics this PR is comparable to `main` but rendering this here moves a lot of modules from async to the main serverlessSearch chunk :/ 

I'm just not sure exactly why yet.